cdc/sink/ddlsink/mysql/mysql_ddl_sink.go

```diff
@@ -136,11 +136,34 @@ func (m *DDLSink) execDDLWithMaxRetries(ctx context.Context, ddl *model.DDLEvent
 		retry.WithIsRetryableErr(errorutil.IsRetryableDDLError))
 }
 
+// isReorgOrPartitionDDL returns true if given ddl type is reorg ddl or
+// partition ddl.
+func isReorgOrPartitionDDL(t timodel.ActionType) bool {
+	// partition related ddl
+	return t == timodel.ActionAddTablePartition ||
+		t == timodel.ActionExchangeTablePartition ||
+		t == timodel.ActionReorganizePartition ||
+		// reorg ddls
+		t == timodel.ActionAddIndex ||
```

Comment:
ActionAddPrimaryKey is also a reorg ddl.
ref: https://github.com/pingcap/tidb/blob/master/ddl/ddl_worker.go#L1051-L1054

Reply:
Thanks for reminder. Added